openssl3/Makefile

```diff
@@ -0,0 +1,172 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2009, 2010, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2021 Joyent, Inc.
+#
+
+VER = openssl-3.0.0
+LIBVER =	3
+
+include ../Makefile.defs
+
+SHELL =		/bin/bash
+BUILD64 =	yes
+UNPACK_SENTINEL =	Configure
+AUTOCONF_OUT =	crypto/opensslconf.h
+
+MT_CPPFLAGS =	-D_REENTRANT
+SHARED_CFLAGS =	-fPIC -DPIC
+SHARED_LDFLAGS =	-fPIC -shared
+
+# XXX KEBE SAYS consider losing the PKCS#11 engine, given 3.0 deprecates it in
```

Comment:
And here?

Reply:
Here I've made a decision, so I need to clarify.  Will fix.